Cargo.toml

```diff
@@ -18,6 +18,7 @@ members = [
     "pallets/claim",
     "primitives/hp-verifiers",
     "primitives/hp-groth16",
+    "verifier-nostd-check",
```

Comment:
Can we put this in the [exclude] section ? Otherwise it gets compiled every time.